packages/icon-build-helpers/src/builders/vanilla/svgo.js

```diff
@@ -171,9 +171,6 @@ const plugins = [
   {
     cleanupNumericValues: true,
   },
-  {
-    moveElemsAttrsToGroup: true,
```

Comment:
Random, what does this option do?

Reply:
Just added it to my note, it does the reverse of what the one below it is trying to do. I think this is why weren't able to properly strip the unused group tags.